### ISSUE TITLE
Exception Details

### ISSUE THREAD

**Issue by kburgoyne** [MicrosoftDocs/winrt-api#1072]
"Otherwise … this method fails and throws an exception."

Need details about what exceptions are thrown when.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ee5f9867-2800-7596-24d1-3e46069c352f
* Version Independent ID: af51891d-f411-fea2-6b73-8d40e72abbf5
* Content: [StorageFile.MoveAsync Method (Windows.Storage) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.storage.storagefile.moveasync?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(Windows.Storage.StorageFile.MoveAsync)%3Bk(TargetFrameworkMoniker-.NETCore%2CVersion%3Dv5.0)%3Bk(DevLang-csharp)%26rd%3Dtrue)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.storage.storagefile.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.storage.storagefile.yml&version=GBlive&_a=contents)
* Product: **uwp**
* Technology: **winrt-reference**
* Git Login: **knicholasa**
* Microsoft Alias: **nichola**

**Comment by groovykool** [comment]
What he said.

**Comment by alvinashcraft** [comment]
Thanks for submitting your feedback and we apologize for the delayed response. I see that updates have not been made since the creation of this Issue, so I went ahead and created an internal work item to investigate.

I created the exception by getting a list of folders grouped by type:

```
var queryOptions = new QueryOptions(CommonFolderQuery.GroupByType);
var query = KnownFolders.PicturesLibrary.CreateFolderQueryWithOptions(queryOptions);
IReadOnlyList<StorageFolder> folders = await query.GetFoldersAsync();
```

When I try to do `StorageFile.MoveAsync` into one of these folders, which isn't a physical folder on disc, I get a `System.Exception` with this message:

`Value does not fall within the expected range.`

It's not a very helpful message, but I can update the docs for this page to let developers know what to expect from the exception.

I will update and close this Issue after the work item is complete to update the docs.
